tests/datasource/fluent/data_asset/data_connector/test_s3_data_connector.py

```diff
@@ -751,150 +746,3 @@ def check_sameness(prefix, expected_output):
     check_sameness("a.x/b/c", "a.x/b/c/")
     check_sameness("path/to/folder.something/", "path/to/folder.something/")
     check_sameness("path/to/folder.something", "path/to/folder.something")
-
-
-@pytest.mark.aws_deps
-@mock_s3
-def test_s3_checkpoint_run_using_different_store_prefixes_successfully(
```

Comment:
REWRITE - uses checkpoint to test stores so we should mock or find another way to test